cmd/juju/commands/enableha.go

```diff
@@ -66,8 +66,9 @@ type enableHACommand struct {
 
 const enableHADoc = `
 To ensure availability of deployed applications, the Juju infrastructure
-must itself be highly available.  enable-ha must be called
-to ensure that the specified number of controllers are made available.
+must itself be highly available. The enable-ha command will ensure
+that the specified number of controller machines are used to make up the
+controller.
```

Comment:
As per IRC, I'm pretty sure that "controller" is supposed to refer to the cluster (if any).